app/views/items/_category_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>
+  <%= form_for Category.new, html: { class: 'form-inline' } do |f| %>
```

Comment:
@joeyjoejoejr - would this be the right type of guide to read? Just wanna make sure I understand what you're referencing. http://nithinbekal.com/posts/rails-presenters/